com.unity.ml-agents.extensions/Runtime/Match3/Match3Actuator.cs

```diff
@@ -134,12 +137,22 @@ public BuiltInActuatorType GetBuiltInActuatorType()
             return BuiltInActuatorType.Match3Actuator;
         }
 
+        /// <inheritdoc/>
         public void Heuristic(in ActionBuffers actionsOut)
         {
             var discreteActions = actionsOut.DiscreteActions;
             discreteActions[0] = GreedyMove();
         }
 
+        /// <summary>
+        /// Returns a valid move that gives the highest value for EvalMovePoints(). If multiple moves have the same
+        /// value, one of them will be chosen with uniform probability.
+        /// </summary>
+        /// <remarks>
+        /// By default, EvalMovePoints() returns 1, so all valid moves are equally likely. Inherit from this class and
+        /// override EvalMovePoints() to use your game's scoring as a better estimate.
+        /// </remarks>
+        /// <returns></returns>
```

Comment:
Should this be protected? I think it should probably be internal, and keep EvalMovePoints() as protected.

Reply:
Updated.